INSTALL.md

```diff
@@ -0,0 +1,59 @@
+# Installationsguide för Baga Aqua System Webbsida
+
+### 2019-05-22
+
+# Förord
+Denna Installationsguide är för Baga Aqua Systems Webbsida. Du förväntas ha installerat Baga Aqua Systems API först. Arbetet är utfört av studenter på Blekinge Tekniska Högskola som en del i kursen PA1416 under vårenterminen 2019 åt Baga Water Technology AB. Denna guide är till för att underlätta vid installation av systemet. Läsaren förväntas ha kunskaper om den egna miljön som installationen sker i.
```

Comment:
First you refer to the reader as "du" and then as "läsaren". Change either of them to the other.
If "du" to "läsaren", change all the remaining "du" in the document.

Reply:
fixed in commit c6f48b8